Branch-SDK/src/io/branch/search/BranchListContentConnection.java

```diff
@@ -8,42 +8,31 @@
 import android.os.RemoteException;
 import android.util.Log;
 
-import java.util.List;
-
 import io.branch.indexing.BranchUniversalObject;
-import io.branch.referral.Branch;
-import io.branch.roots.Roots;
 
 /**
  * Created by sojanpr on 9/26/16.
  * <p>
  * Service connection class for BranchSearchService.
- * Implements  {@link IBranchSearchServiceInterface} methods implemented by BranchSearchService.
+ * Implements  {@link IBranchListContentInterface} methods implemented by BranchSearchService.
  * </p>
  */
-public class BranchSearchServiceConnection implements ServiceConnection {
-    private static BranchSearchServiceConnection connection_;
-    IBranchSearchServiceInterface branchSearchServiceInterface_;
-    private SearchBuilder.IBranchSearchEvent searchEvents_;
-    private RecommendationBuilder.IRecommendationEvents recommendationEvents_;
+public class BranchListContentConnection implements ServiceConnection {
+    private static BranchListContentConnection connection_;
+    IBranchListContentInterface branchSearchServiceInterface_;
     String packageName_;
 
-    public static BranchSearchServiceConnection getInstance() {
+    public static BranchListContentConnection getInstance() {
         if (connection_ == null) {
-            connection_ = new BranchSearchServiceConnection();
+            connection_ = new BranchListContentConnection();
         }
         return connection_;
     }
 
     @Override
     public void onServiceConnected(ComponentName name, IBinder service) {
         Log.d("Bridge_test", "onServiceConnected()");
```

Comment:
If you do put them behind a debug flag, we need more details in the log message than just the function name.